packages/file_selector/file_selector_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.1+7
+
+* Updates to  Pigeon 13.
```

Comment:
same double space issue here

Reply:
But definitely not in all of the other update PRs too 😇